### PR TITLE
Added reponse headers to HubSpotException in order to detect 502

### PR DESCRIPTION
**Pull request by Nazgaul**
Sometime i get 502 from hubspot.
In the current exception -its not easy to detect this.

### CODE REVIEW

**Comment by nover** [comment]
Hi @Nazgaul - thanks for the contribution. I've been on vacation but I'm back now.

File: src/HubSpotException.cs [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
 using System.Collections.Generic;
+using System.Net.Http;
 using System.Text;
 
 namespace Skarp.HubSpotClient
 {
     [Serializable]
     public class HubSpotException : Exception
     {
+        private HttpResponseMessage response;
```
Comment (nover) [inline src/HubSpotException.cs:11]: it would probably be better to have this as a public read-only property to make it accessible for consumers ?

**Comment by Nazgaul** [comment]
You are right.
Change the fields on  HubSpotException to get only field.

**Comment by nover** [comment]
@Nazgaul your change has just been released as package version `v0.9.0`

**Pull request merged**